modules/runner/src/integrationTest/java/org/apache/ignite/internal/runner/app/jdbc/ItJdbcMetadataSelfTest.java

```diff
@@ -204,13 +205,34 @@ public void testGetColumns() throws Exception {
 
         ResultSet rs = meta.getColumns("IGNITE", "PUBLIC", "PERSON", "%");
 
+        checkPersonTableColumns(rs);
+
+        rs = meta.getColumns(null, "PUBLIC", "PERSON", null);
+
+        checkPersonTableColumns(rs);
+
+        rs = meta.getColumns("IGNITE", "PUBLIC", "ORGANIZATION", "%");
+
+        checkOrgTableColumns(rs);
+
+        rs = meta.getColumns(null, "PUBLIC", "ORGANIZATION", null);
+
+        checkOrgTableColumns(rs);
+    }
+
+    /**
+     * Checks organisation table column names and types.
+     *
+     * @param rs ResultSet.
+     * */
+    private void checkOrgTableColumns(ResultSet rs) throws SQLException {
         assertNotNull(rs);
 
-        Collection<String> names = new ArrayList<>(2);
+        List<String> names = new ArrayList<>();
```

Comment:
Collection - you don`t need ordered List here, just for code clarification with equal functionality in : checkPersonTableColumns

Reply:
Fixed, tnx.